bin/xml/ts.top.import.xml

```diff
@@ -34,6 +34,11 @@
     <import file="./ts.vehicles.xml"/>
     
     <!-- PROPERTIES -->
+    <condition property="noSecurityManager" value="true">
+        <javaversion atleast="18"/>
```

Comment:
Is there a reason that anyone would want to run with a Security Manager here?  Remember that this is only changing the JavaTest harness process itself, and has zero impact on how the prospective CI is configured and behaves.  This could only affect sameJVM testing, which I don't believe to be valid for our TCK, so I think we could all literally run with the noSecurityManager property set in all of our runs (with any JDK) and see no difference in our ability to run the tests successfully.

Reply:
> Is there a reason that anyone would want to run with a Security Manager here? 

I haven't experimented with sameJVM for many years (via native code wrappers that launch the JVM) so it does seem unlikely that I would experience the same problem but others could.

I didn't realize that this pr is for a non-general case but still, I don't understand why you wouldn't want the flexibility to enable the SecurityManager with SE 18 when sameJVM is being used.  That sounds likely to vary between different EE implementations.

Reply:
The PR is for a general case.  Everyone will hit the exception when running with Java 18 when trying to use JavaTest, but the SecurityManager currently used here is only actually useful in the sameJVM case per the comments in the JavaTest class.

My understanding was that the sameJVM mode is not a valid way to be running the Platform TCK, but I suppose we would need clarification on that point from someone who knows better than I.  It's not a matter of not wanting flexibility, it's a matter of whether said flexibility should be a requirement if no one is ever going to use it.  There's useful flexibility, and then there's flexibility for flexibility's sake.

Reply:
That makes sense so we only impacting the case of the Java Test Harness Main entry point use of Java Security Manager.